app/views/catalog/access_panels/_online.html.erb

```diff
@@ -6,6 +6,7 @@
 %>
 
 <div class="panel panel-default access-panel panel-online <%= visible %>">
+  <h2>Online</h2>
```

Comment:
@camillevilla, I put this inside the panel that gets displayed/hidden as appropriate w/ the Google links.  Maybe I was miss-understanding the issue, so please let me know if I've missed something.

Reply:
Huh. Looking back at the original design ticket, I may have taken "Add `<h2>Online</h2>` above `panel-online`" too literally 😅